test/onnx/exporter/test_small_models_e2e.py

```diff
@@ -4,6 +4,9 @@
 from __future__ import annotations
 
 import logging
+from unittest import skipIf as skipif
```

Comment:
```suggestion
from unittest import skipIf
```

nit: redundant aliasing